cmd/serve/grant.go

```diff
@@ -182,7 +193,6 @@ func setupRouter(ctx context.Context, logger *zerolog.Logger) (context.Context,
 
 	r.Get("/health-check", handlers.HealthCheckHandler(version, buildTime, commit))
 
-	env := os.Getenv("ENV")
 	reputationServer := os.Getenv("REPUTATION_SERVER")
```

Comment:
Should make these cobra flags/viper bindings?

Reply:
was thinking it was best to leave this for another pr, but happy to do so now

Reply:
Makes sense, here is an issue: https://github.com/brave-intl/bat-go/issues/543